packages/ndla-ui/src/Gloss/Gloss.tsx

```diff
@@ -0,0 +1,163 @@
+import { useTranslation } from 'react-i18next';
+import { colors, spacing, misc, fonts } from '@ndla/core';
+import { AccordionRoot, AccordionItem, AccordionHeader, AccordionContent } from '@ndla/accordion';
+import { Transcription } from '../model/Transcriptions';
+import SpeechControl from '../AudioPlayer/SpeechControl';
+
+interface Example {
+  example: string;
+  language: string;
+  transcriptions: Transcription;
+}
+
+export interface Props {
+  title: {
+    title: string;
+    language: string;
+  };
+  glossData: {
+    gloss: string;
+    wordClass?: string;
+    originalLanguage: string;
+    transcriptions: Transcription;
+    examples?: Example[][];
+  };
+  audio: {
+    title: string;
+    src?: string;
+  };
+}
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  background-color: ${colors.background.lightBlue};
+  padding: ${spacing.nsmall} ${spacing.normal};
+  border: 1px solid ${colors.brand.lighter};
+  border-radius: ${misc.borderRadius};
+  margin-bottom: ${spacing.xsmall};
+  gap: ${spacing.nsmall};
+`;
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+`;
+
+const GlossContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: ${spacing.nsmall};
+`;
+
+const GlossSpan = styled.span`
+  font-weight: ${fonts.weight.bold};
+`;
+
+const TypeSpan = styled.span`
+  font-style: italic;
+`;
+
+const StyledAccordionHeader = styled(AccordionHeader)`
+  font-family: ${fonts.sans};
+  ${fonts.sizes('16px', '24px')};
+  font-weight: ${fonts.weight.semibold};
+  background-color: ${colors.background.lightBlue};
+`;
+
+const StyledAccordionContent = styled(AccordionContent)`
+  padding: 0;
+`;
+
+const TranslatedText = styled.span`
+  display: flex;
+  flex-direction: column;
+  border-bottom: 1px solid ${colors.brand.lighter};
+  padding: ${spacing.small} ${spacing.normal};
+  font-family: ${fonts.sans};
+  ${fonts.sizes('18px', '24px')};
+  :first-child {
+    color: ${colors.brand.dark};
+    font-weight: ${fonts.weight.bold};
+    background-color: ${colors.background.lightBlue};
+  }
+`;
+
+const Gloss = ({ title, glossData, audio }: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Container>
+        <Wrapper>
+          <GlossContainer>
+            <GlossSpan lang={glossData.originalLanguage}>{glossData.gloss}</GlossSpan>
+            {glossData.transcriptions.traditional && (
+              <span
+                key={t(`gloss.transcriptions.traditional`)}
+                aria-label={t(`gloss.transcriptions.traditional`)}
+                lang={glossData.originalLanguage}
+              >
+                {glossData.transcriptions.traditional}
+              </span>
+            )}
+            {glossData.transcriptions.pinyin && (
+              <span
+                key={t(`gloss.transcriptions.pinyin`)}
+                aria-label={t(`gloss.transcriptions.pinyin`)}
```

Comment:
Treng ikkje backticks på disse så lenge du ikkje skal ha inn andre verdier i strengen.
```suggestion
                key={t('gloss.transcriptions.traditional')}
                aria-label={t('gloss.transcriptions.traditional')}
                lang={glossData.originalLanguage}
              >
                {glossData.transcriptions.traditional}
              </span>
            )}
            {glossData.transcriptions.pinyin && (
              <span
                key={t('gloss.transcriptions.pinyin')}
                aria-label={t('gloss.transcriptions.pinyin')}
```